javascript-sdk/etf2l/types/team/teamCompetition.ts

```diff
@@ -0,0 +1,10 @@
+import type { CompetitionResultsDivision } from "../competition/competitionResultsDivision.ts";
+
+export type TeamCompetition = {
+  [competitionId: string]: {
```

Comment:
hm, I think this should be Record<string, Type>

Reply:
I personally much prefer this style since it's more verbose. It kinda tells us what the keys that are returned will be/will reference